spec/fixtures/test_private_key_1.txt

```diff
@@ -0,0 +1,2 @@
8MyIGL4sS+5kjTEosxxnIxmXRQigbwGgUhho41kiQUj22SWqJXwmmyEVB/TV/CkROO2t3640s2eYdcCkLjhfUqUPj04EyiTvnzMZL5/+NZ3s5lVSf4Rml77ruqVNACLp6FY1ovFaeYT6vu0GvUzA1qagoY7RlFLXg83vvSo+RpaXhjnKRXTovbw9eTE3b1P25H08a7XI2e1S9bdFpcVkbtHhuglkoCwwa63qKMv5/McCuTaSg9u6ST8z2i1fsCHYFZQ0lEmvOe9LsI+4zaEU+9C83v0RXke+zTlWZgA/MdG1ejkkiK1QHuDnkmEP1+KTSXsbSM+Lt5kFFeEBlAjQx95eu4ucQY0FmQhHilm6kJYqw461wi1nBlsqPud5xmDHhqOMmlFi/arZL4w2K6HMJoycnquk6Ct6viP4JbA38UU+O8m69jnQxPU7uCL7rrvyXK9PDd3MAH1ckoYXkbVlRtvMFTLy2wcOfNA4b33OkHOsozT3YqaUkKzj5gaC24+KV0+sArvl4VLhskPjyB4JAJHFL+h9mdAVwmds416rJJh1q8zOc/bPqLe1itz6b8gsn5K03rVDLm76mk9n/RVWPU0b58LdnXZ7AAhFfhXU/VvPmUrtoDaEUEwKjQKDmcseoj9qnLsLoqcoxfPaBrlYzyYKvsesiqpPiCfTMX1bTKuP/Wef8LOkR/Ddj0vrJiS1jTSuOZDcVp9P7KQp+5OtZVjuLRWlsLkXltFR2vsf0Bvgo8u0GSfaKXJdiOEJlGvr6mYNBNewt77qpGHeuHn8fVyHUazre/1E509ZJhTsFpA9KqrkRBtAp6ucGxq4wXBblOv8eWnpxEi4pnT88ev4fbFM4EzCJkUKOxk0tjWBPJlAVqrkcEYyZdx+esOd0h/vVbRQx8pzniQtS3tGs81VHJytBHLEfLXSla7FMmrM1bhBSXH4vymycAiuPBLHeF8vNoqOCIQkJ5V4H33nAzLdsBYUfEU5efYja4XPEYUKcQskGamdzQwTl7bczTvVycT2yRE7Vf9NSBaI36XmWXmfY5K6fpJtP/m7VvT12zhyVkuTZ3/Yuz4TeyoFnk5D2Ff4N2FJJ0Bu6HUnc/vI9L1FaAysAEo7BLBTU3A6/iRIb6RXUJrOuxRX3rEiEYkctIfiRZf3ySeH5EAD8z17UF20z+d6M1h4iI7lDN0iU6MbxIYTtKHWw/d6BVQk7QdadlhkbvE7yYTYQucd4A00nwY0wy1nMmEu9UvJNBmjVnmS1Y7nuehjjiCZdCvDeBXsggYCJvbk8vUGC3VMjj1dxOaGj/xp6boeEUw/PBWtfWh61uA6UE9X+m70u4O6/qxShHKfrqcjD3HxAEUuc29mUPqANksItCwBtSyf+bTt8rQ31GyfY2x3r6vj4F8gpLx7b+2QXXS80DuRM1oYkseOkpgywdfwio3S/lWcK7yGruC0/8yh0ttKxFddlNJnXK0HmUZHfbIJ2EP3O9lr4okvGihEKp6ZWy6DCgeimkbJKdBgN+dAne0OoCpj27S9Qz1u5NUp+2hHgdz22Ml8k4W1DMNALWSwKXbpUK6u4htTtZ0SIkf8b2BJBOBjwVH88wMkE3aTlwPeQP19A6MSPeeXEiBf/j0i0rjZZ4FeQ5QyS0WbIh9O/3Ms0rxKgW9wdz5+40Ry0VdO8CR9uq/6xzqKPBicPI/+reI0vOXXjsC95koyLO+/KUURVV7EAtaUz/tOd0LWq0iuNmiuOhbbtaq1/eZh8QZ13Slrgi6MB4VFXhxtat1jPUatJLRTcUSAo06Injk/NQ4defY4j0N1+kasP76rZLR7ImPZjlCmOih/Ys0vKMSPfXZEKYW4X3jSk6U0X8OGaOUvEd+lS0NvzBvZkCheq9N+X0DpvnLw7lAPvAkpGAJuD92u41TdtzhNL+bP6KrPjKLWTeN3H/Oxh0l9k2yDh0/QG+QKKDt/AzRPAtgCo3qEO7wEKGGFA5pGozOfzGtW5+NJ7gdE9HNdrEbQkZ+Sx0p24o8c7PsuKfiqcXgXXyYuAPBXqhdfKSrqyV4TeM60eY6v1ZYnK7dvXQGXhP5kEzsu+flFLe3jBInY1AILq0ERWOP6Qc6k7me7ifiaf/jcGPn1YDE6YzhFDiNLeR25V2X+SdC8J20EZ3oqNFp3GmTDXtoVCexSvd+3dKP9PkLxMlPG9idz+sdLXj56olufSuk4ApxVqhdUacQdikXS9THvMVJu1dcatv5wazORTLgdzY2tKqCl3dSy0vcXapQKU1LGr/YckqQLtHQcjo6N6LxZbiVQ0iHd7lNQNWODD0i4dTBosboTFmUTHpv6Lv4uKJBZOTz3aV1GEYYYkFXKlW1TUQ83i7IXGmUc2OwEPlYqtsfFctfgfincNIedhyNv7x8nNKljfJInMKv8wLbVBzfIBOMdcOB91ux7c8ChY5vEqRRypbdkpQvbZ4UkRbmOfFKjygtpwxD/cmRBqJ6Wuq9FFSQeFMtRSE3QUVeYFy3vqU2xI04ZTEDyqO+qitiISLgVHkQrhBYGV+zqzb21Br4PSSkt8XWrK190/+PIo05jC0Gm1otd7wUuvibA1EFPto0mE8r/jI4LvfsxXVhPOgVEhWkb8bcME4FqmMNr6fwC2xPYnDPULvY9pKoLivkshwqC+CwgIrwZiscnJNraNVsm119VIorPNGEuC0AA4C0tnHn4O+Fj/nk9qoIElDP3HEvNCLoVb+7cQoPPS0XulOX3bg8J4qedEtLOKeB2+8hMfiM1gEERyhYUBHiiMUPlfgZQQj7lGGWBNrCP/WIt73a+F9vtud+g1LnoUyAOj7mX3tsxMOUi7U68DsRr6IUinAE1YY+sNSZMAc2YE32NIcZpCtvac8YL6nagvsLiedaHRHuWpEU1b2Lt7EXdvSx72iBec+aD6LYhHDEl6BOuEhgd0eOhs01O1sU2mDMk2ZDjHufNR7dMM2B4KtNXjtwi581ITgwK7B7jpSkURy/9QvvG2cZxU+ho8="}
```

Comment:
(non-blocking) in the past I've found it helpful to generate keypairs as part of the test setup - think that's worth considering changing up here?

Reply:
The problem is that all of the key pair generation is done clientside :(, and we don't want to have to navigate and go through all that for every single test. Maybe in future we can make a nicer way to do this, but for now this is a good start to get system specs at least started for this repo.